UsefulProteomicsDatabases/UsefulProteomicsDatabases.csproj

```diff
@@ -19,4 +19,10 @@
     <ProjectReference Include="..\Proteomics\Proteomics.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="UniProtKB_columnNamesForProgrammaticAccess.txt">
```

Comment:
Need to add this to the nuspec file

Reply:
Like this: https://github.com/smith-chem-wisc/mzLib/blob/master/mzLib.nuspec#L36

Reply:
does this look right? hard to tell outside of metamorpheus:
![image](https://user-images.githubusercontent.com/16841846/93028125-76e95b80-f5d7-11ea-864c-340785656eca.png)

Reply:
I guess I'm a little confused because I'd expect it to copy to the directory `UsefulProteomicsDatabases`, in which case I think it should be `netcoreapp3.0\UsefulProteomicsDatabases\UniProtKB_column...` and the target should be `lib/netcoreapp3.0/UsefulProteomicsDatabases`. Does it copy to the base directory when you build it?

Reply:
Just tested the nuget packaging, and it works.